### PR TITLE
fixed bug in parallel greedy policy

### PR DESCRIPTION
**Pull request by weepingwillowben**
Ok, I found a real bug this time. ParallelGreedyPolicy had a randn() instead of a rand() when comparing against episilon.

### CODE REVIEW

File: all/policies/greedy.py [hunk 1]
```diff
@@ -70,13 +70,13 @@ def __init__(
     def __call__(self, state):
         best_actions = torch.argmax(self.q(state), dim=-1)
         random_actions = torch.randint(0, self.n_actions, best_actions.shape, device=best_actions.device)
-        choices = (torch.randn_like(best_actions) < self.epsilon).int()
+        choices = (torch.rand(best_actions.shape, device=best_actions.device) < self.epsilon).int()
```
Comment (cpnota) [inline all/policies/greedy.py:73]: Why not rand_like?

**Review by cpnota — COMMENTED** [review]
Ah, good catch.

File: all/policies/greedy.py (hunk 1 above)
Comment (weepingwillowben) [reply all/policies/greedy.py:73]: No particular reason, I just thought it should be consistent.

Comment (cpnota) [reply all/policies/greedy.py:73]: That is a fair point. Why don't we use rand_like and randint_like?

Comment (weepingwillowben) [reply all/policies/greedy.py]: So I changed the rand to rand_like. randint_like is a bit awkward to use, so I just left it.

Comment (cpnota) [reply all/policies/greedy.py]: Ah shoot, this creates some random other issue. I guess just put it back the way you had it, sorry. I'll approve. You can squash & merge once the CI passes.

**Review by cpnota — APPROVED** [review]

**Review by cpnota — APPROVED** [review]

**Pull request merged**